DevelopedCalculator.js

```diff
@@ -16,33 +17,29 @@ class DevelopedCalculator extends Calculator {
         }
         // new method has a parameter
     onButtonNumberClick(event) {
-        // calling parent class's onButtonNumberClick method
+        // calling parent class's onButtonNumberClick method1
         super.onButtonNumberClick(event);
         // pushing the currentResult to the history array 
-        this.history.push(this.currentResult);
+        // this.history.push(this.currentResult);
         // calling gettingHistoryArray to update the history element after clicking any number button 
         this.gettingHistoryArray();
     }
     gettingHistoryArray() {
         // updating the history element by the history which is joined with ,
-        //  this.historyElement.textContent = this.history.join(",");
-        // updating the history element by the history which is joined with ""
-        this.historyElement.textContent = this.history.join("");
+        this.historyElement.textContent = this.history.join(",")
     };
-
     onOperatorButtonClick(event) {
-        // calling parent class's onOperatorButtonClick method
-        super.onOperatorButtonClick(event);
-        // pushing the operator to the history array 
-        this.history.push(this.operator);
-        // calling gettingHistoryArray to update the history element after clicking any operator button 
-        this.gettingHistoryArray();
-    }
-    onClearButtonClick() {
-        super.onClearButtonClick()
-    }
+            // calling parent class's onOperatorButtonClick method
+            super.onOperatorButtonClick(event);
+            const curntresultandopr = this.currentResult + this.operator
```

Comment:
here you're pushing the current result with the operator 

so the history div will show something like 

123456789+ 

we want it to show
123456789,+